apps/expert/lib/expert.ex

```diff
@@ -51,19 +52,33 @@ defmodule Expert do
 
   def handle_request(%GenLSP.Requests.Initialize{} = request, lsp) do
     state = assigns(lsp).state
-    Process.send_after(self(), :default_config, :timer.seconds(5))
```

Comment:
Note (same as before): the logic behind `:default_config` is noop since we always initialize the `state.configuration`. I removed this and related lines.